From 548efe5333b56a0a042843a044e0348317f80065 Mon Sep 17 00:00:00 2001
From: Arthur de Jong <arthur@arthurdejong.org>
Date: Mon, 11 Nov 2019 23:06:20 +0100
Subject: [PATCH] Log the correct timeout value

This fixes logging of the LDAP_OPT_TIMEOUT, LDAP_OPT_NETWORK_TIMEOUT and
LDAP_X_OPT_CONNECT_TIMEOUT options to actually log the value of the
bind_timelimit option instead of the timelimit option.
---
 nslcd/myldap.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/nslcd/myldap.c b/nslcd/myldap.c
index 02b97195..0ee1be80 100644
--- a/nslcd/myldap.c
+++ b/nslcd/myldap.c
@@ -911,17 +911,17 @@ static int do_set_options(MYLDAP_SESSION *session)
   tv.tv_usec = 0;
 #ifdef LDAP_OPT_TIMEOUT
   log_log(LOG_DEBUG, "ldap_set_option(LDAP_OPT_TIMEOUT,%d)",
-          nslcd_cfg->timelimit);
+          nslcd_cfg->bind_timelimit);
   LDAP_SET_OPTION(session->ld, LDAP_OPT_TIMEOUT, &tv);
 #endif /* LDAP_OPT_TIMEOUT */
 #ifdef LDAP_OPT_NETWORK_TIMEOUT
   log_log(LOG_DEBUG, "ldap_set_option(LDAP_OPT_NETWORK_TIMEOUT,%d)",
-          nslcd_cfg->timelimit);
+          nslcd_cfg->bind_timelimit);
   LDAP_SET_OPTION(session->ld, LDAP_OPT_NETWORK_TIMEOUT, &tv);
 #endif /* LDAP_OPT_NETWORK_TIMEOUT */
 #ifdef LDAP_X_OPT_CONNECT_TIMEOUT
   log_log(LOG_DEBUG, "ldap_set_option(LDAP_X_OPT_CONNECT_TIMEOUT,%d)",
-          nslcd_cfg->timelimit);
+          nslcd_cfg->bind_timelimit);
   LDAP_SET_OPTION(session->ld, LDAP_X_OPT_CONNECT_TIMEOUT, &tv);
 #endif /* LDAP_X_OPT_CONNECT_TIMEOUT */
   log_log(LOG_DEBUG, "ldap_set_option(LDAP_OPT_REFERRALS,%s)",
